cli/Valet/Configuration.php

```diff
@@ -42,6 +42,13 @@ function install()
     function createConfigurationDirectory()
     {
         $this->files->ensureDirExists(VALET_HOME_PATH, user());
+
+        $oldPath = posix_getpwuid(fileowner(__FILE__))['dir'].'/.valet';
+
+        if ($this->files->isDir($oldPath)) {
+            rename($oldPath, VALET_HOME_PATH);
+            Nginx::rewriteSecureNginxFiles();
```

Comment:
This `Nginx::rewriteSecureNginxFiles();` line can be removed, since running `valet install` will already fire it a couple steps later.